packages/znc/no_pthread.patch

```diff
@@ -0,0 +1,22 @@
+--- znc-1.8.2/src/main.cpp	2020-09-08 05:57:50.000000000 +0700
++++ znc-1.8.2.mod/src/main.cpp	2022-02-21 17:08:57.974706825 +0700
+@@ -180,7 +180,6 @@
+         m_thread = std::thread([=]() { HandleSignals(pZNC); });
+     }
+     ~CSignalHandler() {
+-        pthread_cancel(m_thread.native_handle());
```

Comment:
Please justify the removal of `pthread_cancel()` call. Doesn't this cause hang up in any cases? Or do we need some workaround as in the `calcurse` package?

Reply:
Thinking about it. Maybe i will do some workaround as in `calcurse` package.